Paraswap-Agents/Role-Changes/src/utils.ts

```diff
@@ -0,0 +1,54 @@
+import { LogDescription, Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export const SWAPPER_CONTRACT = "0xDEF171Fe48CF0115B1d80b88dc8eAB59176FEe57"; //AugustusSwapper contract
+export const EVENTS_ABI = [
+  "event RoleAdminChanged(bytes32 indexed role, bytes32 indexed previousAdminRole, bytes32 indexed newAdminRole)",
+  "event RoleGranted(bytes32 indexed role, address indexed account, address indexed sender)",
+  "event RoleRevoked(bytes32 indexed role, address indexed account, address indexed sender)",
+];
+
+export const createFinding = (log: LogDescription) => {
+  if (log.name == "RoleAdminChanged") {
+    return Finding.fromObject({
+      name: "Admin role change detected on AccessControl contract",
+      description: `${log.name} event emitted`,
```

Comment:
For all findings generated it seems more appropriate for the `name` to be swapped with `description`.
Typically the `name` is a short explanation with `description` going into a bit more detail.
For the findings in this agent it seems to be the opposite.
```suggestion
      name: `${log.name} event emitted`,
      description: "Admin role change detected on AccessControl contract",
```